jira7/jira_data_mappings.bal

```diff
@@ -21,7 +21,7 @@ import ballerina/config;
 import ballerina/mime;
 
 function errorToJiraConnectorError(error source) returns JiraConnectorError {
-    JiraConnectorError target = source.message != EMPTY_STRING ? {message:source.message, cause:source.cause} : {};
+    JiraConnectorError target = source.message != EMPTY_STRING ? {message:source.message} : {};
```

Comment:
Shall we include the cause instead of dropping it? We can do it as follows.
```ballerina
JiraConnectorError target = source.message != EMPTY_STRING ? {message:source["message"],cause:source["cause"]} : {};
```